src/styles/main.scss

```
@@ -0,0 +1,120 @@
+$watch-size: 400px;
+$pin-size: $watch-size/30;
+
+@mixin watch-hand($w, $h, $clr, $br-clr, $spin-dur) {
```

```suggestion
@mixin watch-hand($w, $h, $clr, $br-clr, $spin-dur) {
```

Крута реалізація, але ось тут людина зі сторони не одразу зрозуміє що ти мав на увазі під цими змінними. Ну або я такий, не виключаю))